docs/System-Requirements.md

```diff
@@ -4,7 +4,13 @@ The Operator is validated for deployment on Kubernetes, GKE and EKS clusters.
 The Operator is cloud native and storage agnostic, working with a wide variety
 of storage classes, hostPath, and NFS.
 
-The Operator was developed and tested with PostgreSQL versions 12.14, 13.10, 14.7, and 15.2. Other options may also work but have not been tested. The Operator provides connection pooling based on [`pgBouncer`](https://www.pgbouncer.org/) 1.18.0 and high-availability implementation based on [`Patroni`](https://patroni.readthedocs.io/en/latest/) 3.0.1.
+## Supported versions
+
+The Operator {{ release }} is developed, tested and based on:
+
+* PostgreSQL 12.14, 13.10, 14.7, and 15.2 as the database. Other versions may also work but have not been tested. 
```

Comment:
```suggestion
* PostgreSQL 12.14, 13.10, 14.7, and 15.2 as the database (other versions may also work but have not been tested)
```

Reply:
I got rid of the parentheses explicitly. According to the style guide one should avoid using them where possible (https://developers.google.com/style/parentheses)

Reply:
Well, internal list item ending with a period visually breaks the list. We have options not to end list items with a punctuation mark, or to put a comma in all paragraphs except the last one, and a period in the last one. If you put a period somewhere not at the end of the list, this conflicts with both schemes